### PR TITLE
Added Agile quiz question

### PR DESCRIPTION
**Pull request by prasad-333**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

issue#103

### CODE REVIEW

**Comment by prasad-333** [comment]
Added 2 more agile questions.

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -290,6 +290,36 @@ const agileQuiz = [
...
+    Link: "https://scrumguides.org/scrum-guide.html"
+  },
+  {
+    Question: "Who is accountable for establishing Scrum and Scrum Team's effectiveness?",
+    Answer: "The Scrum Master",
+    Distractor1: "The Scrum Team",
+    Distractor2: "The Product Owner and the Scrum Master",
+    Distractor3: "The Product Owner",
+    Explanation:
+      "Scrum Master is responsible for establishing scrum within team & at organization level. They achieve this by enabling the Scrum Team to improve its practices, within scrum framework and help them understand Scrum theory & practices.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:311]: I think it is fine to leave off the last sentence. If users want to learn more, then they can always click on the learn more link in the quiz. 
```suggestion
      " The Scrum Master is responsible for establishing scrum within the team & organizational level.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**